geometric_kernels/lab_extras/extras.py

```diff
@@ -20,3 +40,29 @@ def from_numpy(_: B.Numeric, b: Union[List, B.Numeric, B.NPNumeric]):
     """
     Converts the array `b` to a tensor of the same backend as `a`
     """
+
+
+@dispatch
+@abstract()
+def swapaxes(a: B.Numeric, axis1: int, axis2: int) -> B.Numeric:
+    """
+    Interchange two axes of an array.
+    """
+
+
+@dispatch
+@abstract()
+def copysign(a: B.Numeric, b: B.Numeric) -> B.Numeric:
+    """
+    Change the sign of `a` to that of `b`, element-wise.
+    """
+
+
+def isclose(a: B.Numeric, b: B.Numeric, rtol=1e-5, atol=1e-8):
+    """Returns a boolean array where two arrays are element-wise equal within a
+    tolerance."""
```

Comment:
Minor: consistency in docs formatting
```suggestion
    """
    Returns a boolean array where two arrays are element-wise equal within a
    tolerance.
    """
```